# HG changeset patch
# User Rik <rik@octave.org>
# Date 1409583511 25200
#      Mon Sep 01 07:58:31 2014 -0700
# Node ID f7c524595a71d4995a89c123247629f3f52f5f9c
# Parent  d20c81d3cd21c7039d4715fcf216502610683bca
test.m: Fix typo in __signal_fail name.

* test.m: Drop 's' from __signal_fails.

diff --git a/scripts/testfun/test.m b/scripts/testfun/test.m
--- a/scripts/testfun/test.m
+++ b/scripts/testfun/test.m
@@ -454,17 +454,17 @@ function [__n, __nmax, __nxfail, __nskip
           __patstr = ifelse (__iswarning, "a warning", "an error");
         endif
       endif
       try
         eval (sprintf ("function __test__(%s)\n%s\nendfunction",
                        __shared, __code));
       catch
         __success = false;
-        __msg = [__signal_fails "test failed: syntax error\n" lasterr()];
+        __msg = [__signal_fail "test failed: syntax error\n" lasterr()];
       end_try_catch
 
       if (__success)
         __success = false;
         __warnstate = warning ("query", "quiet");
         warning ("on", "quiet");
         ## Clear error and warning strings before starting
         lasterr ("");
